test/test/helpers/use-i18n/test/index.js

```diff
@@ -1,32 +0,0 @@
-/**
```

Comment:
It would be a nice idea to keep this integration test in place :)

In theory it should work as before after updating import statements and removing before block.